portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,53 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Tianan Li's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script.js"></script> 
   </head>
-  <body>
+  <body background="images/background.jpeg">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 style="text-align:center">This is Tianan Li's portfolio</h1>
+      <hr>
+      <h2>Who I am</h2>
+      <div>
+      <div class="photo"><img style="max-width:80%;overflow:hidden;text-align:center;" src="images/photo.jpg"></div>
```

Comment:
An overall comment for this whole file about formatting: could you indent the html sub-elements?

Reply:
Thanks for your suggestions. Fixed.